hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -849,7 +867,14 @@ public HapiSpec creationValidatesName() {
                         recordSystemProperty("tokens.maxTokenNameUtf8Bytes", Integer::parseInt, maxUtf8Bytes::set))
                 .when()
                 .then(
-                        tokenCreate(PRIMARY).name("").logged().hasPrecheck(MISSING_TOKEN_NAME),
+                        tokenCreate(PRIMARY)
+                                .name("")
+                                .logged()
+                                // Either fail pre-check with MISSING_TOKEN_NAME (mono)
+                                // or post-consensus (modular). When we only have modular code left,
+                                // we can just check hasKnownStatus.
+                                .hasPrecheckFrom(OK, MISSING_TOKEN_NAME)
+                                .hasKnownStatus(MISSING_TOKEN_NAME),
```

Comment:
Same here. Instead of this you can add MISSING_TOKEN_NAME to this property `spec.streamlinedIngestChecks`. This will avoid modifying tests